packages/form/src/components/SwitchInput.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { InputSwitch } from "primereact/inputswitch";
+import { UseFormGetFieldState, UseFormRegister } from "react-hook-form";
+
+import { ErrorMessage } from "./ErrorMessage";
+interface ISwitch {
+  checked: boolean;
```

Comment:
we probably don't need this prop. This should be handled by register method by default. Please check.